sdks/python/apache_beam/utils/options.py

```diff
@@ -488,6 +488,24 @@ def _add_argparse_args(cls, parser):
          '(--staging_location option) and the workers will install them in '
          'same order they were specified on the command line.'))
 
+
+class TestOptions(PipelineOptions):
+
+  @classmethod
+  def _add_argparse_args(cls, parser):
+    # Options for e2e test pipeline.
+    parser.add_argument(
+        '--on_success_matcher',
```

Comment:
If you plan on creating one eventually, start by creating a JIRA issue.

What is `onCreateMatcher` used in Java, we can verify the same things here.